sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,242 @@
+# July'21 release notes
+
+Release notes for commit range 6a49170027fb..962909fe9e78
+
+## New features
+ - Implemented SYCL 2020 specialization constants [07b27965] [ba3d657]
```

Comment:
should we mention the SYCL 2020 Exception support and/or the sRGB extension?

Reply:
There are not included in the July release from github. The release includes everything up to 962909fe9e78bd597594d67c5c5be7eacf8fb10a.